src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -71,6 +73,22 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
+        public static void Close() {
+            var folder = GetTempCsvFilesFolder();
+            if (Directory.Exists(folder)) {
+                foreach (var f in Directory.EnumerateFiles(folder)) {
+                    try {
+                        File.Delete(f);
+                    } catch(IOException) { } catch (UnauthorizedAccessException) { }
+                }
+            }
+        }
+
+        private static string GetTempCsvFilesFolder() {
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
I think you're confusing "Export to CSV" with "Open in Excel". They are two different commands, and this particular code only pertains to the latter. If the user wants a file on disk that they want to open later, they should use "Export".

In the case of "Open", writing that data to file is an implementation detail (we didn't even do that before, but loading it directly into Excel sheet via COM APIs proved to be too slow). That file is going to be automatically deleted when VS is closed, so there's no "Excel centric processing post RTVS session" that can happen here.